lapce-proxy/src/lsp.rs

```diff
@@ -84,15 +85,51 @@ impl LspCatalog {
         language_id: &str,
         options: Option<Value>,
     ) {
+        let binary_args = self.get_plugin_binary_args(options.clone());
         let client = LspClient::new(
             language_id.to_string(),
             exec_path,
             options,
+            binary_args.to_owned(),
```

Comment:
Why do you call `to_owned` on a `Vec`? If you need a copy of it, call `clone()` instead, it's cleaner. Otherwise, just pass ownership as I don't see any other use of the variable.